### PR TITLE
ZEP-2239: Add "Void Payout" to NZ API Doc

### PR DESCRIPTION
**Pull request by dylan-zepto**
## 👀 Reviewers

* simple change just see it’s sane

## 💡 Why
* [ZEP-2239](https://zeptoau.atlassian.net/browse/ZEP-2239)

## 📝 What

- Added Void a Payment to NZ API Doc
- Excluded [deprecated] Retry a Prefailed Payment Request

## 🏚 Before
image

## 🏡 After
image

### CODE REVIEW

**Review by SmileAndCompile — APPROVED** [review]
Nice Job!

File: source/openapi3/split.yaml [hunk 1]
```diff
@@ -1505,6 +1505,22 @@ tags:
...
+  - name: Payouts
+    description: >
+      This endpoint gives you some control over a transaction:
+
+
+      * After it has been created; and
+
+      * Before it has been submitted to the banks; or
+
+      * If it was Prefailed, due to insufficient funds, and was therefore not submitted to the banks.
```
Comment (corroded) [inline source/openapi3/split.yaml]: @henripryde400 @chrisixion do we have prefail in NZ?

Comment (dylan-zepto) [reply source/openapi3/split.yaml]: Line removed as per standup discussion this morning

File: source/index.html.md [hunk 2]
```diff
@@ -4429,6 +4429,187 @@ Get a single payment by its reference
 |---|---|---|---|
 |200|[OK](https://tools.ietf.org/html/rfc7231#section-6.3.1)|OK|[GetAPaymentResponse](#schemagetapaymentresponse)|
 
+<h1 id="Zepto-API-Payouts">Payouts</h1>
```
Comment (friendlyantz) [inline source/index.html.md:4432]: I noted you mix various naming conventions for html id's in this PR - 
```html
# see line 4432
<h1 id="Zepto-API-Payouts">Payouts</h1>

# vs 4446
<a id="opIdVoidAPayment"></a>

```

do we have convention for `id`'s? this is a rabbithole discussion, but some interesting points are [outlined here](https://stackoverflow.com/questions/6028211/what-is-the-standard-naming-convention-for-html-css-ids-and-classes)

is having `-` might pose a risk / ambiguity when working with `js`?
perhaps `zeptoApiPayouts` or `zepto_api_payouts` would be a better alternative? 

worth RFC @corroded, or this was agreed already?

File: source/index.html.md [hunk 3]
```diff
@@ -4429,6 +4429,187 @@ Get a single payment by its reference
...
+|---|---|---|---|---|
+|ref|path|string|true|Payment debit reference number e.g D.48|
+
+<h3 id="Void a Payment-responses">Responses</h3>
+
+|Status|Meaning|Description|Schema|
+|---|---|---|---|
+|204|[No Content](https://tools.ietf.org/html/rfc7231#section-6.3.5)|No Content|None|
+
 <h1 id="Zepto-API-Refunds">Refunds</h1>
```
Comment (friendlyantz) [inline source/index.html.md:4613]: same as other comments

File: source/index.html.md [hunk 4]
```diff
@@ -4429,6 +4429,187 @@ Get a single payment by its reference
...
+* Before it has been submitted to the banks; or
+
+<aside class="notice">
+  Payments and Payment Requests are made up of individual Debits and Credits. These debits and credits
+  were once referred to as Payouts [Legacy naming].
+</aside>
+
+## Void a Payment
+
+<a id="opIdVoidAPayment"></a>
```
Comment (friendlyantz) [inline source/index.html.md:4446]: Different convention FYI

**Review by friendlyantz — APPROVED** [review]

File: source/index.html.md (hunk 2 above)
Comment (corroded) [reply source/index.html.md:4432]: this is autogenerated so we don't need to worry about it

**Pull request merged**